addons/nachschub/configs/CfgVehicles.hpp

```diff
@@ -193,11 +195,11 @@ class CfgVehicles
                         displayName = "Raktenwerfer Kisten";
                         icon = "A3\ui_f\data\map\diary\icons\unitPlayable_ca.paa";
 
-                        ADD_SUPPLY("Werfer",TB_supply_usa_launcher);
```

Comment:
Bin ich eigl dagegen, da in manchen Mission Missionbauer einfach allegemein was geben wollen. Zudem ggf in großen Missions eine allgemeine Mission. Müssen wir nochmal drüber reden.

Reply:
Ja, können wir gerne machen.
Kann denn nicht mit mehreren Kisten diesbezüglich ausgeholfen werden? Und die Spieler suchen sich dann aus, was sie brauchen? Ist kein Beinbruch, wieder die allgemeinen Kisten reinzupacken und zusätzlich die Spezialisierten drin zu lassen.
Ich bin mir nur unsicher, dass es Szenarien gibt, in denen zwei allgemeine Kisten besser sind als zwei unterschiedlich Spezialisierte.
- allgemeine Kiste = wenig passende Munition für den einzelnen Werfer = mehrere allgemeine Kisten notwendig
- zwei unterschiedliche Werfer/MGs in der Mission (Trupp A = MWAAS/M240 ; Trupp B = SMAW/M249) :
    - pro Trupp je eine allgemeine Kiste = Hälfte der Kiste von jedem Trupp nur zu gebrauchen = gemeinsamer 
      Sammelpunkt notwendig, um die Kisten auszutauschen
    - pro Trupp je eine spezialisierte Kiste = keine Interaktion zwischen Trupps nötig, dafür situativ mehr Munition
    - Stinger/Javelinmunition bis auf wenige Missionen pro Jahr sowieso nicht in allgemeiner Werferkiste notwendig, da 
      beides nicht zusätzlich mitgenommen wird, sondern nur wenn Missionsbauer es anmeldet (darf sich gerne ändern, 
      dass es häufiger im Fahrzeug bspweise mitgeführt wird - darum auch meine Anfrage letzte Woche an dich bezüglich 
      Fahrzeug Inventar zu Missionsbeginn)